fdbserver/VersionedBTree.actor.cpp

```diff
@@ -7548,7 +7569,11 @@ TEST_CASE("/redwood/correctness/unit/deltaTree/IntIntPair") {
 	std::set<IntIntPair> uniqueItems;
 	while (uniqueItems.size() < N) {
 		IntIntPair p = randomPair();
-		if (uniqueItems.count(p) == 0) {
+		auto nextP = p; // also check if next highest/lowest key is not in set for testLTE/testGTE
```

Comment:
A clearer explanation about the assumptions about the key pattern would be useful. i.e. please clarify that the keys are supposed to be non-consecutive, not just unique.

Reply:
I added a couple comments that should help clarify.